ihatemoney/web.py

```diff
@@ -848,6 +850,32 @@ def settle_bill():
     return render_template("settle_bills.html", bills=bills, current_view="settle_bill")
 
 
+@main.route("/<project_id>/settle/<amount>/<int:ower_id>/<int:payer_id>")
+def settle(amount, ower_id, payer_id):
+    # FIXME: Test this bill belongs to this project !
+    # form = get_billform_for(g.project, set_default=False)
+    # form.bill_type = ("Refund", "Refund")
+    # form.amount = amount
+    # form.payer = ower
+    # form.payedfor = payer
+
+    new_reinbursement = Bill(
+        amount=float(amount),
+        date=datetime.datetime.today(),
+        owers=[Person.query.get(payer_id)],
+        payer_id=ower_id,
+        project_default_currency=g.project.default_currency,
+        bill_type="Reimbursement",
+        what="Settlement",
```

Comment:
We might want this to be translated?

Reply:
I agree! If I understand correctly, to make this string translatable I just need to wrap it with `babel.gettext`, or `_`?

Reply:
Alright, done!